bite_keeper/bite_all_keeper.py

```diff
@@ -85,9 +181,30 @@ def check_cup(self, cup_id, axe: Ray, par: Ray, tag: Ray):
 
         # Bite cups with owe over a threshold that haven't been bitten before
         if owe > Ray.from_number(0) and cup.art != Wad.from_number(0):
-            self.logger.info(f'Bite cup {cup_id} with owe of {owe} and ink of {cup.ink}')
+            self.logger.info(
+                f'Bite cup {cup_id} with owe of {owe} and ink of {cup.ink}'
+            )
             self.tub.bite(cup_id).transact(gas_price=self.gas_price())
 
+    def call_bitecdps(self):
+        self.logger.info(f'Will bite top {self.top} CDPs')
+        # cdps = [i+1 for i in range(self.tub.cupi())]
+        cdps = self.get_cdps()
+        self.logger.info(f'found {len(cdps)} CDPs')
+        for i in range(0, len(cdps), self.chunks):
+            chunk = cdps[i:i+self.chunks]
+            self.logger.info(f'BiteCdps.bite({chunk})')
+            self.bitecdps.bite(chunk).transact(gas_price=self.gas_price())
+
+    def get_cdps(self):
+        client = GraphQLClient(self.graphql_url)
+        result = client.execute(QUERY.replace('XXX', f'{self.top}'))
+        data = json.loads(result)
+        cdps = []
+        for cdp in data["data"]["allCups"]["nodes"]:
+            cdps.append(cdp["id"])
+        return cdps
+
     def gas_price(self):
```

Comment:
I lightly recommend utilizing [GeometricGasPrice](https://github.com/makerdao/pymaker/blob/master/pymaker/gas.py#L144), as it ensures the next update in gas price is above Parity's limit for replacement transactions. 

```
GWEI = 1000000000

return GeometricGasPrice(initial_price=5*GWEI,
                         every_secs=30,
                         max_price=200*GWEI)
```